test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -131,6 +131,26 @@ def test_severities
     end
   end
 
+  def test_struct_payload_non_utf8_log
+    setup_gce_metadata_stubs
+    setup_logging_stubs do
+      d = create_driver
+      d.emit('msg' => log_entry(0),
+             'normal_key' => 'test' + non_utf8_character + 'some non utf8',
+             'non_utf8' + non_utf8_character + 'key' => 5000,
+             'some_null_field' => nil)
```

Comment:
Do we also want to test the case where a non-utf8 character appears nested in a structured payload?